docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -35,7 +35,10 @@ Excel add-ins run across multiple versions of Office, including Office 2016 or l
 | [ExcelApi 1.1](excel-api-1-1-requirement-set.md)  | Version 1509 (Build 4266.1001) or later   | 1.19 or later | 15.20 or later | January 2016 |
 
 > [!NOTE]
-> The build number for Office 2016 installed via MSI is 16.0.4266.1001. This version only contains the ExcelApi 1.1 requirement set.
+> Users on perpetual versions of Office will be restricted to certain requirement set.
```

Comment:
Consider phrasing this in terms of what perpetual versions of Office support (rather than what users have access to). i.e., maybe something like -> Perpetual versions of Office support requirement sets as follows: